content/manager/security.md

```diff
@@ -181,12 +181,36 @@ class name. <br>
 * properties - a dictionary of arguments required to instantiate the implementing class. The arguments will be passed as
 kwargs to the class' `__init__` method.
 
+#### Simple Userstore driver
 The default configuration uses [Flask-SecuREST's simple userstore]
 (https://github.com/cloudify-cosmo/flask-securest/blob/0.7/flask_securest/userstores/simple.py).
-In this implementation, the userstore is a simple data-structure defined in the manager blueprint and instantiated at
-bootstrap. As shown in the configuration above, it contains a single user, and reads its username and password from
-input values (`admin_username` and `admin_password` respectively). This implementation is good for demonstrations
+In this implementation, the userstore is a simple data-structure defined in the manager blueprint and instantiated during
+bootstrap.
+
+**Configuring the Simple Userstore:**
+
+* implementation - The implementation field should point to `flask_securest.userstores.simple:SimpleUserstore`. <br>
+* properties - Holds the actual userstore to be used as demonstrated in the example above.
+{{% gsNote title="Note" %}}
+The configuration above describes a userstore having a single user, that accepts its username and password from
+input values (`admin_username` and `admin_password` respectively). This implementation is good for demonstration
 purposes. Configuring a "real" userstore driver (e.g. ActiveDirectory) is just as easy, as explained later.
+{{% /gsNote %}}
+
+#### File Userstore driver (As of Flask-SecuREST 0.8)
+The file based userstore provides the ability to define users and groups in an external file located on the manager.
+This file can later be edited and will be reloaded by the Flask-SecuREST framework upon modification. A sample userstore file can be found [here]
```

Comment:
is the file read for each authentication/authorization? or does it watch for changes?